ConsoleApp1/Point.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace ConsoleApp1
+{
+    class Point
+    {
+        private int _x;
+        private int _y;
+
+        public int X { get { return _x; } set { _x = value; } }
+        public int Y { get { return _y; } set { _y = value; } }
+
+        public Point(int x, int y)
+        {
+            _x = x;
+            _y = y;
+        }
+        public override bool Equals(object? obj)
```

Comment:
better to implement `IEquatable` and reuse it in the override, C# is pretty clever and will use IEquatable implementation whenever Equals called